circle.yml

```diff
@@ -17,18 +17,14 @@ checkout:
   post:
     - ./scripts/circle-ci/pull-all-images.sh:
         background: true
-    - ./scripts/circle-ci/stop-services.sh
-
-general:
-  artifacts:
-    - "build/reports/profile"
+    - ./scripts/circle-ci/stop-services.sh:
+        background: true
```

Comment:
Is it fine if some services are stopped after tests start running? Background doesnt guarantee that this will be complete before. There are two background scripts now, are both run in parallel, pull-images is likely to take a long time.